### DISCUSSION TITLE
Bookmarks filters and ordering

### DISCUSSION THREAD

**Discussion by dicarsmtz** [dailydotdev/daily#1414] [Feature Requests]
Would be awesome to have a way to filter bookmarks at least by their name and to allow us to sort them from the oldest bookmarked.

I've been using bookmarks as a place where I store articles I like to read, but I would like to keep adding more while I read the oldest ones and take them out of the list, but currently I have to scroll all the way down

**Comment by idoshamun** [comment]
Hi @amdaor,

Let's keep the discussion here: https://github.com/dailydotdev/daily/discussions/520
